terra_notebook_utils/drs.py

```diff
@@ -259,6 +259,8 @@ def copy_to_bucket(drs_url: str,
     if dst_bucket_name is None:
         dst_bucket_name = WORKSPACE_BUCKET
     src_client, src_info = resolve_drs_for_gs_storage(drs_url)
+    if not dst_key:
+        dst_key = src_info.name or src_info.key.rsplit("/", 1)[-1]
```

Comment:
Could also be `dst_key = dst_key or src_info.name or src_info.key.rsplit("/", 1)[-1]` .  Pretty minor line saved though, so I don't think it's important.

Reply:
Could be! Leaving as is.